api/dump_meta_data.md

```diff
@@ -4,11 +4,14 @@ To help when asking for support and reporting bugs,
 TimescaleDB includes a SQL script that outputs metadata
 from the internal TimescaleDB tables as well as version information.
 The script is available in the source distribution in `scripts/`
-but can also be [downloaded separately][].
+but can also be [downloaded separately][scripts-download].
```

Comment:
@ryanbooz Where can the `dump_meta_data` script be downloaded? The link isn't actually in this file.

Reply:
@ryanbooz just bumping this, if you can't remember or don't want to deal with it now, I can just remove the link for now